packages/lb-annotation/src/core/toolOperation/basicToolOperation.ts

```diff
@@ -675,6 +675,13 @@ class BasicToolOperation extends EventListener {
     this.forbidCursorLine = false;
   }
 
+  public clearCursorLine() {
```

Comment:
What's diff between forbidCursorLine and clearCursorLine ?

Reply:
`ForbidCursorLine` is always not showing  cursorLine. But `ClearCursorLine` just move coordinate out of view to hidden cursorLine